cinema/views.py

```diff
@@ -21,24 +28,52 @@
 )
 
 
-class GenreViewSet(viewsets.ModelViewSet):
+class GenreViewSet(mixins.ListModelMixin,
+                   mixins.CreateModelMixin,
+                   viewsets.GenericViewSet):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
 
+    def get_permissions(self):
+        if self.action == "create":
+            return [IsAdminUser()]
+        return [IsAuthenticated()]
+
 
-class ActorViewSet(viewsets.ModelViewSet):
+class ActorViewSet(mixins.ListModelMixin,
+                   mixins.CreateModelMixin,
+                   viewsets.GenericViewSet):
     queryset = Actor.objects.all()
     serializer_class = ActorSerializer
 
+    def get_permissions(self):
+        if self.action == "create":
+            return [IsAdminUser()]
+        return [IsAuthenticated()]
```

Comment:
The `get_permissions` method in `ActorViewSet` restricts the `create` action to admins only. As per the task requirements, both list and create actions should be accessible to all users. Consider adjusting the permissions to allow all users to create.